tests/bootstrap.php

```diff
@@ -0,0 +1,21 @@
+<?php declare(strict_types=1);
+
+require_once __DIR__ . '/../vendor/autoload.php';
+
+$pimcoreProjectRootDir = __DIR__ . '/app';
+$_SERVER['PIMCORE_PROJECT_ROOT'] = $pimcoreProjectRootDir;
+$_ENV['PIMCORE_PROJECT_ROOT'] = $pimcoreProjectRootDir;
+putenv('PIMCORE_PROJECT_ROOT=' . $pimcoreProjectRootDir);
+
+$kernelClass = 'TestKernel';
+$_SERVER['KERNEL_CLASS'] = $kernelClass;
+$_ENV['KERNEL_CLASS'] = $kernelClass;
+putenv('KERNEL_CLASS=' . $kernelClass);
+
+$appEnv = 'test';
+$_SERVER['APP_ENV'] = $appEnv;
+$_ENV['APP_ENV'] = $appEnv;
+putenv('APP_ENV=' . $appEnv);
```

Comment:
Not sure if we actually need to set it everywhere (probably just `$_SERVER` is enough), but if we do, I'd create a little helper like:

```suggestion
$setEnv = function (string $name, string $value): void {
    putenv("{$name}=".$_ENV[$name] = $_SERVER[$name] = $value);
};

$setEnv('PIMCORE_PROJECT_ROOT', __DIR__ . '/app');
$setEnv('KERNEL_CLASS', 'TestKernel');
$setEnv('APP_ENV', 'test');
```

Reply:
You are right. I just could copy the helper function, you used before :)

Reply:
That's what I did in the comment :wink: